source/common/router/rds_impl.cc

```diff
@@ -97,7 +98,7 @@ Router::ConfigConstSharedPtr RdsRouteConfigProviderImpl::config() {
 
 void RdsRouteConfigProviderImpl::onConfigUpdate(const ResourceVector& resources,
                                                 const std::string& version_info) {
-  last_updated_ = factory_context_.systemTimeSource().currentTime();
+  last_updated_ = ProdSystemTimeSource::instance_.currentTime();
```

Comment:
Have a TimeSource& as member variable would be slightly better.

Reply:
Yeah, I need to mock it for the `//test/common/router:rds_impl_test` anyway.